package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-rdkafka",
-  "version": "v2.7.4",
+  "version": "v3.0.0",
```

Comment:
I bumped the package major because this is a breaking change, but I can revert it that's not how its done here.

Reply:
Is this breaking change because versions 8 and older are dropped or is there some API breaking change?

Reply:
breaking because it drops support 8.x and older, no API change